assets/common-css/navbar.css

```diff
@@ -0,0 +1,33 @@
+
+.navbar-container{
+    padding: 20px;
```

Comment:
Prefer to use `rem` for larger than 10px 
and also better to use CSS variables, like you have for the below code as well